packages/fuels-test-helpers/src/chains.rs

```diff
@@ -1,18 +1,15 @@
-use crate::{utility, Opcode, Transaction, REG_ONE};
+use crate::utils;
 use fuel_types::AssetId;
-use fuels_core::constants::{
-    BASE_ASSET_ID, DEFAULT_BYTE_PRICE, DEFAULT_GAS_LIMIT, DEFAULT_GAS_PRICE, DEFAULT_MATURITY,
-    DEFAULT_SPENDABLE_COIN_AMOUNT,
-};
-use fuels_core::errors::Error;
-use fuels_core::tx::{Input, Output};
+use fuels_core::constants::{BASE_ASSET_ID, DEFAULT_SPENDABLE_COIN_AMOUNT};
+use fuels_core::tx::{Input, Output, Transaction};
 use fuels_signers::provider::Provider;
 use fuels_signers::{LocalWallet, Signer};
+use fuels_types::errors::Error;
 use rand::Rng;
 use std::time::Duration;
-use utility::retry_until;
+use utils::retry_until;
 
-/// Can be used to generate `amount` number of blocks. A block is generated by
+/// Can be used to produce `amount` number of blocks. A block is produced by
 /// submitting a no-op script for execution. Transactions are submitted one
 /// after the other. A new transaction won't be submitted until an increase of
 /// block height has been noticed.
```

Comment:
I'm not a fan of creating a ticket that just refers the `fuel-core` ticket unless you think it's really necessary. Doesn't have to be a `TODO`, but we need to document that it *is* a workaround anyway.

Reply:
The thing I would like to achieve is for us to remember to improve this once the fuel core ticket is done.

AFAICS we can do that by either doing a TODO (which is kind of annotated in the IDE) but that has a habit of staying around without ppl noticing. 

Or we can create an issue like 'use the new tx pool API for producing blocks' which will remind us to do it by its very presence in the backlog.

Because, if I'm not mistaken, the only reminder we have now is our short-term memory and the notification you're going to get if somebody starts working on the fuel core task since you're the reporter.

Reply:
> I'm not a fan of creating a ticket that just refers the fuel-core ticket unless you think it's really necessary. Doesn't have to be a TODO, but we need to document that it is a workaround anyway.

What better way to document it than having a ticket for it, TODOs without a ticket are useless

Reply:
What I mean is we already have a ticket for it in `fuel-core`. Possibly we can add in that ticket that the change should also be ported to the SDK, but I don't see the merit in having duplicate tickets accross repos.

Reply:
The ticket in `fuel-core` is for them to provide an API. 

The ticket in `fuels-rs` would be for us to use it.

So they are related but not duplicated, but I understand the feeling. 

I'll create a ticket in fuels-rs, if you don't mind, just to have some peace of mind otherwise it's going to be taking up space as a sticky note on the wall of my office.

Reply:
@segfault-magnet PR is open for `fuel-core` if you want to have the ticket tracking that [#461](https://github.com/FuelLabs/fuel-core/pull/461). You could wait for that to go through but idk how concerned you are about this getting stale and it may be easier to just merge this PR then use the new API in a later one